react-native-cli/index.js

```diff
@@ -221,8 +221,15 @@ function checkNodeVersion() {
 
 function checkForVersionArgument() {
   if (process.argv.indexOf('-v') >= 0 || process.argv.indexOf('--version') >= 0) {
-    var pjson = require('./package.json');
-    console.log(pjson.version);
+    var cliJson = require('./package.json');
+    console.log('react-native-cli: ' + cliJson.version);
```

Comment:
Extra variable seems unnecessary, you can just do `console.log('react-native-cli: ' + require('./package.json').version)`